modules/login/php/endpoints/authentication.class.inc

```diff
@@ -0,0 +1,453 @@
+<?php declare(strict_types=1);
+/**
+ * POST request methods.
+ *
+ * Used to retrieve info with authentication by the login form.
+ *
+ * PHP Version 7
+ *
+ * @category LORIS_Module
+ * @package  Login
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\login\endpoints;
+
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+use \LORIS\Middleware\ETagCalculator;
+use \LORIS\api\Endpoint;
+use SinglePointLogin;
+use State;
+
+/**
+ * The NDB_Page for the POST requests.
```

Comment:
The function comment need to be updated/cleaned up to match the functionality of this file.